server/src/main.rs

```diff
@@ -441,7 +442,7 @@ mod tests {
     use crate::handle_request;
     use async_std::task::block_on;
     use http_types::{Method, Request, StatusCode, Url};
-    use oxigraph::RocksDbStore;
+    use oxigraph::Store;
```

Comment:
Sadly, it does not work because Rust is looking for `Store` in the `oxigraph` crate directly.

I believe this works:
```suggestion
    use super::Store;
```

Reply:
Note to self: always run the tests yourself. ;)